vectorvis/integrallinefiltering/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+#--------------------------------------------------------------------
+# Inviwo IntegralLineFiltering Module
+ivw_module(IntegralLineFiltering)
+
+#--------------------------------------------------------------------
+# Add header files
+set(HEADER_FILES
+    include/inviwo/integrallinefiltering/integrallinefilteringmodule.h
+    include/inviwo/integrallinefiltering/integrallinefilteringmoduledefine.h
+    include/inviwo/integrallinefiltering/processors/integrallinestodataframe.h
+    include/inviwo/integrallinefiltering/utils/directionalhistogram.h
```

Comment:
We should try hard not to use util... this should probably be in datastructues
and the entropy in algorithm